modules/osdk-control-compat.adoc

```diff
@@ -76,13 +77,8 @@ com.redhat.openshift.versions: "v4.6-v4.8" <1>
 +
 [source,yaml]
 ----
-LABEL com.redhat.openshift.versions="v4.6-v4.8" <1>
+LABEL com.redhat.openshift.versions="<versions>" <1>
 ----
-<1> Set to a range or single version.
-+
-[NOTE]
-====
-This label is also useful when you know that the current version of your Operator will not work well, for any reason, on a specific {product-title} version. By using it, you define the cluster versions where the Operator should be distributed, and the Operator does not appear in a catalog of a cluster version which is outside of the range.
-====
+<1> Set to a range or single version, for example, `v4.6-v4.8`.
```

Comment:
```suggestion
<1> Set to a range or single version, for example, `v4.6-v4.8`.  By using it you defined the Openshift versions where the Operator should be distributed and the Operator will not appear in a catalog of an Openshift version that is outside of the range. You must use it if you are distributing a solution that contains deprecated API(s) and will no longer be available in later versions.
```

WDYT about we improve the desc here to make it more clear?

Reply:
Added:

> This setting defines the cluster versions where the Operator should be distributed, and the Operator does not appear in a catalog of a cluster version which is outside of the range.

I think the additional suggested sentence "You must use it if you are distributing a solution that contains deprecated API(s) and will no longer be available in later versions" has already been said a few times earlier by this point so I think it would be redundant to say again?

Reply:
It just to clarifies but you are right.